components/autocomplete_selector.tsx

```diff
@@ -58,12 +79,12 @@ export default class AutocompleteSelector extends React.PureComponent {
 
         requestAnimationFrame(() => {
             if (this.suggestionRef) {
-                this.suggestionRef.blur();
+                this.suggestionRef.current?.blur();
             }
         });
     }
 
-    setSuggestionRef = (ref) => {
+    setSuggestionRef = (ref: React.RefObject<HTMLDivElement>) => {
```

Comment:
```suggestion
    setSuggestionRef = (ref: RefObject<HTMLDivElement>) => {
```
nit Import from react ⏫